hrm-domain/lambdas/custom-integrations/uscr-beacon-poller/tests/service/dbConnection.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import pgPromise from 'pg-promise';
+
+export const pgp = pgPromise({});
+
+export const db = pgp(
+  `postgres://hrm:postgres@127.0.0.1:${
+    process.env.HRM_DATABASE_PORT ?? '5432'
+  }/hrmdb?&application_name=service-test-beacon-poller`,
```

Comment:
Seems strange to have `?&' here? But maybe I misunderstand the syntax.

Reply:
It works but probably should tidy up